macros/utils/percentile.sql

```diff
@@ -1,11 +0,0 @@
-{% macro percentile(percentile_field, partition_field, percent) -%}
-
-{{ adapter.dispatch('percentile_impl', 're_data') (percentile_field, partition_field, percent) }}
```

Comment:
In case of removing this @bachng2017 Trino re-data version: https://github.com/bachng2017/dbt-re-data-trino/blob/main/macros/utils/percentile.sql would need some update (basically removing `impl` from the name). I think it would still be clearer and that's good with me. @bachng2017 Any objections? :)